src/pages/public_registration/LogPanel.tsx

```diff
@@ -149,9 +154,12 @@ export const LogPanel = ({ tickets, registration }: LogPanelProps) => {
           return (
             <StyledStatusMessageBox key={index} sx={{ bgcolor: ticketColor[logItem.type] }}>
               <Typography>{logItem.description}</Typography>
-              <Tooltip title={modifiedDate.toLocaleTimeString()} enterDelay={tooltipDelay}>
-                <Typography>{modifiedDate.toLocaleDateString()}</Typography>
-              </Tooltip>
+              <Box sx={{ display: 'flex' }}>
```

Comment:
Når jeg ser denne tenker jeg det er mer naturlig å ha gap her i stedet for `ml` i Avatar ja
```suggestion
              <Box sx={{ display: 'flex', gap: '0.5rem' }}>
```

Reply:
Gir mening!